Formula/a/ayatana-ido.rb

```diff
@@ -0,0 +1,41 @@
+class AyatanaIdo < Formula
+  desc "Ayatana Indicator Display Objects"
+  homepage "https://github.com/AyatanaIndicators/ayatana-ido"
+  url "https://github.com/AyatanaIndicators/ayatana-ido/archive/refs/tags/0.10.4.tar.gz"
+  sha256 "bd59abd5f1314e411d0d55ce3643e91cef633271f58126be529de5fb71c5ab38"
+  license any_of: ["GPL-3.0-or-later", "LGPL-2.0-or-later"]
+
+  depends_on "cmake" => :build
+  depends_on "gobject-introspection" => :build
+  depends_on "vala" => :build
+  depends_on "pkgconf" => :test
```

Comment:
```suggestion
  depends_on "pkgconf" => [:build, :test]
```
Looks like you need this to fix the build failure on Linux. I suspect you're getting away with it on macOS since `pkgconf` is already installed anyway, and is always in `PATH` when available (unlike on Linux).